src/style-spec/reference/v8.json

```
@@ -3881,10 +3890,13 @@
         ]
       },
       "transition": true,
-      "doc": "",
+      "doc": "The color of space behind the globe and atmosphere.",
```

This entry doesn't solely work with the globe, so I think we should remove the reference to it. Otherwise, that can suggest that this will only work with the globe projection:

```suggestion
      "doc": "The color of the region above the horizon and after the end of the `horizon-blend` contribution. The opacity can be set to `0` to have a transparent background.",
```